src/unittest/test_utilities.cpp

```diff
@@ -636,3 +638,69 @@ void TestUtilities::testSanitizeDirName()
 	UASSERT(sanitizeDirName("cOnIn$", "~") == "~cOnIn$");
 	UASSERT(sanitizeDirName(" cOnIn$ ", "~") == "_cOnIn$_");
 }
+
+template <typename F, typename C>
+C container_map(const C &co, F functor)
```

Comment:
Yes it maps but `map` is also a container which somewhat confused me. What about `static C vector_process_elementwise(....)` ?